README.md

```diff
@@ -61,7 +61,7 @@ Here is a list of community roles with current and previous members:
   - [Fabrizio Ferri-Benedetti](https://github.com/theletterf)
   - [Patrice Chalin](https://github.com/chalin), CNCF
   - [Phillip Carter](https://github.com/cartermp), Honeycomb
-  - [Severin Neumann](https://github.com/svrnm), Cisco
+  - [Severin Neumann](https://github.com/svrnm), Independent
```

Comment:
I don't think we _need_ to specify any affiliation. And I'd aim for consistency with Fabrizio's entry:

```suggestion
  - [Severin Neumann](https://github.com/svrnm)
```